js/main.js

```diff
@@ -511,7 +512,8 @@
 		},
 		
 		/*
-		
+		    send the other side "<you> is recording audio..."
```

Comment:
Capitalise the "Send" please, and if you may, please change <you> to <name> for elegancy.